README.md

```diff
@@ -28,8 +28,8 @@ Setup your computer to accept Gazebo packages from packages.osrfoundation.org.
 ```bash
 sudo apt update
 sudo apt install -y wget
-echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable bionic main" > /etc/apt/sources.list.d/gazebo-stable.list
-wget https://packages.osrfoundation.org/gazebo.key -O - | apt-key add -
+echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable bionic main" > sudo /etc/apt/sources.list.d/gazebo-stable.list
```

Comment:
This needs to be `focal` for 20.04.

Reply:
Updated https://github.com/osrf/rmf_demos/pull/103/commits/55951661bd20e062127b99f8cd5d34fd6bff56a6